angular.json

```diff
@@ -159,7 +159,14 @@
                 "node_modules/@spartacus/styles/scss"
               ]
             },
-            "codeCoverageExclude": ["**/index.ts", "**/*.de.ts", "**/*.en.ts", "**/*store.module.ts"]
+            "codeCoverageExclude": [
+              "**/index.ts",
+              "**/*.de.ts",
+              "**/*.en.ts",
+              "**/*store.module.ts",
+              "projects/fsastorefrontlib/src/core/date-config/default-date-config.ts",
```

Comment:
We don't want to exclude only these 2 configs. Exclude all files that are ending on *config.ts